Sources/AppKit/NSTableView.swift

```diff
@@ -36,20 +37,164 @@ public extension ReactiveExtensions where Base: NSTableView {
   }
 }
 
-public extension SignalProtocol where Element: DataSourceEventProtocol, Element.DataSource: QueryableDataSourceProtocol, Element.DataSource.Item: Any, Element.DataSource.Index == Int, Error == NoError {
+// MARK: - Table view integration
+
+extension NSTableView {
+	
+	fileprivate var updating: Bool {
+		get {
+			return objc_getAssociatedObject(self, &NSTableViewUpdatingKey) as? Bool ?? false
+		}
```

Comment:
Is associating value types valid thing to do? :)

In any case, I think we should move this property into `DefaultTableViewBond`. Of course, it means that `DefaultTableViewBond` becomes a class, but that's ok.